src/plugins/kibana/server/lib/get_mappings.js

```diff
@@ -0,0 +1,39 @@
+const _ = require('lodash');
+const Promise = require('bluebird');
+const {templateToPattern, patternToTemplate} = require('./convert_pattern_and_template_name');
+
+module.exports = function getMappings(pattern, req) {
```

Comment:
Methods that require a request object can generally be refactored to accept one or two more parameters (like a bound version of `callWithRequest` in this instance) and in the process become simpler to test.